### ISSUE TITLE
Raspberry Pi Pico -- BSD chat hangs while attempting to connect with target

### ISSUE THREAD

**Issue by anarchitech** [nimblemachines/muforth#22]
...  and no useful output is available to the user (due to failure of picoboot.mu4 to finish building we don't have spy or bug)

```
./muforth -d testing -f target/ARM/board/raspberry-pi-pico.mu4 
muforth/64 (60d83f19) 2021-nov-13 16:51
(https://muforth.nimblemachines.com/)
Copyright (c) 2002-2021 David Frech (read the LICENSE for details)

Type 'settings' to see a few of muforth's tweakable behaviours.

(( Raspberry Pi Pico board
(( RP2040 chip
(( RP2040 equates ))
(( ARM Cortex-M0+ processor
(( ARM Cortex-M core equates (v6-M and v7-M) ))
(( Target endianness ))
(( ARM v6-M meta-compiler (target-specific chains) ))
(( Meta-compiler (chains and token consumers) ))
(( ARM v6-M memory image ))
(( Reading Motorola S-record (S19) files ))
(( ARM v6-M Thumb2 disassembler ))
(( ARM v6-M Thumb2 assembler 4# again.  00r again.  i8 again.  sh-imm
again.  ))
(( ARM v6-M meta-compiler (main) ; again.  ))
(( ARM v6-M interaction dumps again.  ))
(( Flash programming save-image again.  ))))))
(( Rasberry Pi PICOBOOT support ))
(( ARM v6-M Forth kernel (ITC) ))))
pico  <---- HANGS HERE
```

**Comment by anarchitech** [comment]
If line 111 of target/ARM/debug/picoboot.mu4 is commented out, boot displays version and then hangs.

```

109  : pi.hello  ( - chunk-size)
          picoboot.reset
111     --  2 picoboot.exclusive  ( eject mass storage)
112    pi.version
113    #256 ;  ( so we get correct alignment for reading and writing flash)
ram
```

**Comment by anarchitech** [comment]
We're very close to committing a working solution for OpenBSD. See #24 .

**Comment by anarchitech** [comment]
Fixed by commit 2a9fea0
